src/utils/adapters.js

```diff
@@ -8,11 +8,13 @@ function isAuction(chainName) {
   return chainName.length - suffixLength < auctionLength
 }
 
-export function adaptKeyblock(keyblock) {
+export function adaptKeyblock(keyblock, keyblockDeltaStats = null) {
   if (keyblock) {
     return {
       ...keyblock,
       mined: DateTime.fromMillis(keyblock.time),
+      block_reward: keyblockDeltaStats ? formatAettosToAe(keyblockDeltaStats.block_reward) : null,
```

Comment:
I am not really satisfied with the sudden change of casing without further discussion. There are more ways how to make it consistent. Now the code casing is inconsistent. It may take a long time till we will start using typescript so I don't like the idea. I suggest making it back consistent in the open PR's or creating refactoring issue to make it consistent real soon.

Reply:
@janmichek I disagree this is a change or introducing more inconsistency. These properties are called this way in our application for many months:

![image](https://github.com/aeternity/aescan/assets/46789227/40375964-5a40-4921-bbab-b713e09d8f83)

I'm all for unifying everything to one case but we need to do it everywhere, not selectively

Reply:
I created a dedicated task to unify that: https://github.com/aeternity/aescan/issues/385